### ISSUE TITLE
This project belongs to js project, but it is not recorded

### ISSUE THREAD

**Issue by qianguyihao** [kaxap/arl#6]
<https://github.com/qianguyihao/web>

This project belongs to js project, but it is not recorded.

**Comment by kaxap** [comment]
yep, it ended up as a [NodeJS](https://github.com/kaxap/arl/blob/master/README-Node.md) repo (Web/703)

**Comment by qianguyihao** [comment]
So, what's the difference between `js repo` and `Node repo` in your project?

**Comment by kaxap** [comment]
The Github API request are made with the following URI pattern:

```https://api.github.com/search/repositories?q=language:{lng}&sort=stars&order=desc&page={n_page}&per_page=100```

where `Node` or `JS` go to the `{lng}` placeholder. 
Seems like qianguyihao/web wasn't properly indexed by github? 🤷‍♂️

**Comment by kaxap** [comment]
@qianguyihao there is no "language details" in the theqianguyihao/web repo. Did you change any corresponding settings?

**Comment by earlyraven** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!

**Comment by Asmodaus** [comment]
# $1000 Christmas Airdrop for 10,000 Eligible Users! 🎅🎁

Welcome to the festive $1000 Christmas Airdrop by LayerZero! Get ready to unwrap the joy and claim your share of $ZRO tokens. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

![Claim Now](https://miro.medium.com/v2/resize:fit:720/format:webp/1*SalqMtUhedutBSfrW7-T-A.png)

## Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Christmas Spirit:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Instagram:**
     [Share on Instagram](https://instagram.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&title=LayerZero+Christmas+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero Christmas Airdrop and secure your $ZRO tokens.

Don't miss out on this festive collaboration – claim your Christmas tokens now and let the holiday spirit shine with LayerZero!
